source/MRMesh/MRObjectLoad.cpp

```diff
@@ -191,7 +191,7 @@ Expected<std::shared_ptr<Object>> makeObjectFromMeshFile( const std::filesystem:
     auto objectMesh = std::make_unique<ObjectMesh>();
     objectMesh->setName( utf8string( file.stem() ) );
     objectMesh->setMesh( std::make_shared<Mesh>( std::move( mesh.value() ) ) );
-
+    metrics.holesCount[0] = int(objectMesh->numHoles());
```

Comment:
what if nullptr?

Reply:
better ues `*metrics.holesCount = int(objectMesh->numHoles());` as far as it is not array